core/src/Cabana_AoSoA.hpp

```diff
@@ -405,7 +406,9 @@ class AoSoA
             _num_soa );
         if ( _num_soa > 0 )
             Kokkos::deep_copy(
-                resized_data,
+                Kokkos::subview(
+                    resized_data,
+                    Kokkos::pair<size_type, size_type>( 0, _num_soa ) ),
```

Comment:
This is unnecessary. `resized_data` already has the right size.
https://github.com/ECP-copa/Cabana/blob/08ae3750bf9743511600c4ceef8e3cd67e2b252f/core/src/Cabana_AoSoA.hpp#L404-L406